test/integration/src/test/java/io/pravega/test/integration/RestoreBackUpDataRecoveryTest.java

```diff
@@ -423,8 +463,8 @@ public void testDurableDataLogFail() throws Exception {
         log.info("Segments have been recovered.");
 
         // Start a new segment store and controller
-        this.segmentStoreStarter = startSegmentStore(this.storageFactory, this.dataLogFactory);
-        controllerStarter = startController(this.bookKeeperStarter.bkPort, this.segmentStoreStarter.servicePort);
+        segmentStoreStarter = startSegmentStore(this.storageFactory, this.dataLogFactory, containerCount);
```

Comment:
I don't see these two instances being closed anywhere.

Reply:
closing them in tearDown.